doc/article/en-US/setup-webpack.md

```diff
@@ -85,3 +85,390 @@ Integration tests are performed with [Protractor](http://angular.github.io/protr
   ```shell
   npm run e2e:start
   ```
+
+## [Using Standard Webpack Configuration](aurelia-doc://section/7/version/1.0.0)
+
+1. After downloading skeleton-esnext-webpack from Aurelia github,
+  we need to replace any reference to @easy-webpack with the standard webpack modules.<br/>In `package.json`, remove all modules that start with `@easy-webpack` in `devDependencies`:
+  
+  ```json
+    "@easy-webpack/config-aurelia": "^2.0.1",
+    "@easy-webpack/config-babel": "^2.0.2",
+    "@easy-webpack/config-common-chunks-simple": "^2.0.1",
+    "@easy-webpack/config-copy-files": "^1.0.0",
+    "@easy-webpack/config-css": "^2.3.2",
+    "@easy-webpack/config-env-development": "^2.1.1",
+    "@easy-webpack/config-env-production": "^2.1.0",
+    "@easy-webpack/config-external-source-maps": "^2.0.1",
+    "@easy-webpack/config-fonts-and-images": "^1.2.1",
+    "@easy-webpack/config-generate-index-html": "^2.0.1",
+    "@easy-webpack/config-global-bluebird": "^1.2.0",
+    "@easy-webpack/config-global-jquery": "^1.2.0",
+    "@easy-webpack/config-global-regenerator": "^1.2.0",
+    "@easy-webpack/config-html": "^2.0.2",
+    "@easy-webpack/config-json": "^2.0.2",
+    "@easy-webpack/config-test-coverage-istanbul": "^2.0.2",
+    "@easy-webpack/config-uglify": "^2.1.0",
+    "@easy-webpack/core": "^1.3.2",
+  ``` 
+
+  with the following:
+
+  ```json
+    "aurelia-webpack-plugin": "^1.1.0",
+    "copy-webpack-plugin": "^3.0.1",
+    "html-webpack-plugin": "^2.22.0",
+    "babel-core": "^6.17.0",
+    "babel-loader": "^6.2.5",
+    "babel-polyfill": "^6.16.0",
+    "css-loader": "^0.25.0",
+    "file-loader": "^0.9.0",
+    "html-loader": "^0.4.4",
+    "sourcemap-istanbul-instrumenter-loader": "^0.2.0",
+    "style-loader": "^0.13.1",
+    "url-loader": "^0.5.7",
+  ```
+
+2. Then use the following config:
+
+  * **IMPORTANT**: the following config is for `webpack@2.1.0.beta-23+` (current: `@beta-25`), as from this version, schema validation will be enforced
+  and custom property on config object is no longer allowed. So we will be using `webpack.LoaderOptionsPlugin` to provide
+  some configs for `html-minifier-loader`. If you want to keep using the prefdefined webpack
+  version in the skeleton, move all properties inside `webpack.LoaderOptionsPlugin` instance to export object
+
+  ```js
+    const hasProcessFlag = flag => process.argv.join('').indexOf(flag) > -1;
+    const path = require('path');
+    const webpack = require('webpack');
+    const HtmlWebpackPlugin = require('html-webpack-plugin');
+    const CopyWebpackPlugin = require('copy-webpack-plugin');
+    const AureliaWebpackPlugin = require('aurelia-webpack-plugin');
+    const WebpackMd5Hash = require('webpack-md5-hash');
+    const DefinePlugin = require('webpack/lib/DefinePlugin');
```

Comment:
@bigopon thanks for writing this! I really appreciate the markup.... following your instructions I noticed an error with webpack build: 2.1.0.beta-23 should be 2.1.0-beta.23 #626